kext/FileNVRAM/FileNVRAM.h

```diff
@@ -180,7 +182,7 @@ class FileNVRAM : public IODTNVRAM
     OSDictionary * mNvramMissDict;
     IOCommandGate* mCommandGate;
     OSString*      mFilePath;
-    IOTimerEventSource* mTimer;
+    IONotifier* disk_notifier_start_;
```

Comment:
shoul not be:  `IOTimerEventSource* mTimer = IOTimerEventSource::timerEventSource(this, timeoutOccurred);`
just:
`mTimer = IOTimerEventSource::timerEventSource(this, timeoutOccurred);` ?